Image_process/processImage.py

```diff
@@ -0,0 +1,136 @@
+import os
+
+from PIL import Image
+
+
+class ImageProcess:
```

Comment:
why is this a class? the only shared state is `image_path` which only gets used in one function. This functionality can be split into multiple separate, and more composable functions.